code/drasil-example/Drasil/GlassBR/Unitals.hs

```diff
@@ -170,14 +170,22 @@ sd_min     = mkQuantDef (unitary "sd_min"
 {--}
 
 glassBRSymbols :: [UnitaryChunk]
-glassBRSymbols = [min_thick, sflawParamK, sflawParamM, demand, load_dur,
+glassBRSymbols = [min_thick, sflawParamK, sflawParamM, demand, lRe, nonFactorL, load_dur,
   eqTNTWeight]
 
-min_thick, sflawParamK, sflawParamM, demand, sdx, sdy, sdz, load_dur,
+min_thick, sflawParamK, sflawParamM, demand, sdx, sdy, sdz, lRe, nonFactorL, load_dur,
   eqTNTWeight :: UnitaryChunk
 
 demand      = unitary "demand"      (nounPhraseSP "applied load (demand)")
   lQ pascal Rational --correct Space used?
+-----------------------------inserted by Vajiheh-----------------------------------------
```

Comment:
Please remove these comments - the 'diff' feature of git already makes that clear (i.e. what has been changed and by whom).

Reply:
@JacquesCarette it was addressed with 1cf95d5.